src/scss/gaiden.scss

```diff
@@ -12,9 +11,10 @@
 @import 'tools/mixins/map_deep_get';
 @import 'tools/mixins/generate_button_sizes';
 @import 'tools/mixins/generate_button_types';
+@import 'tools/mixins/get_border';
 @import 'tools/mixins/get_color';
 @import 'tools/mixins/get_shadow';
-@import 'tools/mixins/get_border';
+@import 'tools/mixins/get-space';
```

Comment:
`get-space` não devaria estar com **underscore** para seguir o padrão que estamos usando?

Reply:
@williamcosta yes. I'll fix this now. `get-weight` has the same problem, but we don't saw in older PR.